web/src/engine/websites/HqNow.ts

```diff
@@ -0,0 +1,173 @@
+import { Tags } from '../Tags';
+import icon from './HqNow.webp';
+import { Chapter, DecoratableMangaScraper,Manga,Page,type MangaPlugin } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { FetchGraphQL, FetchRequest } from '../FetchProvider';
+
+type APIMangas = {
+    getHqsByNameStartingLetter: APIManga[],
+};
+
+type APISingleManga = {
+    getHqsById: APIManga[],
+};
+
+type APIManga = {
+    id: number,
+    name: string,
+    capitulos: APIChapter[]
+};
+
+type APIChapter = {
+    name: string,
+    id: number,
+    number: number
+};
+
+type APIPages = {
+    getChapterById: {
+        pictures: { pictureUrl: string }[]
+    }
+};
+
+const apiUrl = 'https://admin.hq-now.com';
```

Comment:
Suggestion to make this a class member

Reply:
can they be used in decorator calls in that case?